downloader/m3u8.go

```diff
@@ -112,14 +130,17 @@ func Download() {
 }
 
 func renameFile() {
-	path1 := filepath.Join(outputPath, "/", "merged.tmp")
-	path2 := filepath.Join(downloadDir, "/", output)
+	path1 := filepath.Join(outputPath, "merged.tmp")
+	path2 := filepath.Join(downloadDir, output)
 	err := os.Rename(path1, path2)
 	if err != nil {
 		log.Println("[error] Rename failed: " + err.Error())
 	}
 	if deleteTS {
-		os.RemoveAll(outputPath)
+		err = os.RemoveAll(outputPath)
+		if err != nil {
+			log.Println(err)
```

Comment:
For me, it's always failed because the temp folder is `use by another process`.

Reply:
Thanks a lot